src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +119,205 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
+		/// </summary>
+		/// <param name="obj">The object that modify the Shell tabs visibility.</param>
+		/// <returns>The navigation animation that occurs when a page is navigated.</returns>
 		public static PresentationMode GetPresentationMode(BindableObject obj) => (PresentationMode)obj.GetValue(PresentationModeProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetPresentationMode']/Docs/*" />
+
+		/// <summary>
+		/// Sets the navigation animation that occurs when a page is navigated to with the <see cref = "GoToAsync(ShellNavigationState, bool)" /> method.
+		/// </summary>
+		/// <param name="obj">The object that modify the Shell tabs visibility.</param>
+		/// <param name="presentationMode">Defines the navigation animation that occurs when a page is navigated.</param>
 		public static void SetPresentationMode(BindableObject obj, PresentationMode presentationMode) => obj.SetValue(PresentationModeProperty, presentationMode);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetFlyoutBehavior']/Docs/*" />
+		/// <summary>
+		/// Gets the navigation animation that occurs when a page is navigated to with the <see cref = "GoToAsync(ShellNavigationState, bool)" /> method.
+		/// </summary>
+		/// <param name="obj">The object that modify the Shell tabs visibility.</param>
+		/// <returns>The navigation animation that occurs when a page is navigated.</returns>
 		public static FlyoutBehavior GetFlyoutBehavior(BindableObject obj) => (FlyoutBehavior)obj.GetValue(FlyoutBehaviorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetFlyoutBehavior']/Docs/*" />
+
+		/// <summary>
+		/// Sets the behavior used to open the flyout.
+		/// The flyout can be accessed through the hamburger icon or by swiping from the side of the screen.
+		/// However, this behavior can be changed by setting the <see cref = "FlyoutBehavior" /> attached property.
+		/// </summary>
+		/// <param name="obj">The object that modify the Shell behavior used to open the flyout.</param>
+		/// <param name="value">The behavior used to open the flyout.</param>
 		public static void SetFlyoutBehavior(BindableObject obj, FlyoutBehavior value) => obj.SetValue(FlyoutBehaviorProperty, value);
 
+		/// <summary>
+		/// Gets the width of the flyout.
+		/// </summary>
+		/// <param name="obj">The object that modify the width of the flyout.</param>
+		/// <returns>The width of the flyout.</returns>
 		public static double GetFlyoutWidth(BindableObject obj) => (double)obj.GetValue(FlyoutWidthProperty);
+
+		/// <summary>
+		/// Sets the width of the flyout.
+		/// This enables scenarios such as expanding the flyout across the entire screen.
+		/// </summary>
+		/// <param name="obj">The object that modify the width of the flyout.</param>
```

Comment:
```suggestion
		/// <param name="obj">The object that modifies the width of the flyout.</param>
```

This suggestion applies to several of the other properties in this section